fvm/bootstrap.go

```diff
@@ -44,6 +44,8 @@ type BootstrapProcedure struct {
 	transactionFees        BootstrapProcedureFeeParameters
 	executionEffortWeights weightedMeter.ExecutionEffortWeights
 	executionMemoryWeights weightedMeter.ExecutionMemoryWeights
+	// executionMemoryLimit of nil means that it won't be set in the state. The FVM will use the default value from the context.
+	executionMemoryLimit *uint64
```

Comment:
I think it would be more go-like to use a value here and compare against default value - 0 rather than use pointer.

Reply:
I think we could use 0 as the "not-set" value since it doesn't make sense to set it to 0

Reply:
[389fb3c](https://github.com/onflow/flow-go/pull/2474/commits/389fb3cc0adbb7727024a2e00e337500c1994894)